### DISCUSSION TITLE
Customize default layer base on country

### DISCUSSION THREAD

**Discussion by julcnx** [openstreetmap/iD#8956] [Ideas]
Bing has been the default layer in ID, and it has been used primarily by new mappers due to a lack of alternative imagery and missing knowledge on how to change layers in editors.

Unfortunately, in Thailand where I map, Bing is often a cause of conflict due to its outdated and misaligned imagery.
We have discussed this in our [forum](https://forum.openstreetmap.org/viewtopic.php?id=74230) and documented alternative recommendations in our [wiki](https://wiki.openstreetmap.org/wiki/WikiProject_Thailand#Aerial_imagery).

However, new mappers will not be aware of these issues and systematically map with Bing.

How could we automatically inform new mappers? some ideas:
- specify a (different) default layer based on current location/country
- refer new mappers/accounts to a list of resources as shown after a commit

What are your thoughts on this and what would be technically feasible ?

**Comment by 1ec5** [comment]
Some previous discussion in osmlab/editor-layer-index#130.
